config.ru

```diff
@@ -0,0 +1,14 @@
+require 'puppet_webhook'
+require 'logger'
```

Comment:
Since you're using `FileUtils` below, you should require it here. Otherwise, you might forget you need it and remove the require from the `puppet_webhook` class.